prime-router/docs/api/README.md

```diff
@@ -62,15 +60,89 @@ Based on OAuth2, this method is based on best practices outlined in the SMART on
 OAuth above: a client secret, ID, and scope is encoded as a JWT and sent to the server; which responds with a bearer
 token the client can use to authenticate and authorize further requests.
 
+The initial JWT must be created outside of the swagger environment. An example using a python script is provided in this
```

Comment:
I think you jump in to eagerly here, I like how you started the Okta section with: 

`OKTA provides an implementation of OAuth2 for authentication.`

but here you assume the reader knows server-server is based on OAuth2 and that they understand what a "JWT" is and how the auth works?